lib/core/lib/ops/userOps.js

```diff
@@ -513,6 +557,43 @@ function buildUserOps (db/*, config */) {
       })
     },
 
+    /**
+     * List the teams to which the user belongs to
+     * Does not go recursively to parent teams
+     *
+     * @param  {Object}   params { id, organizationId, limit, page }
+     * @param  {Function} cb
+     */
+    listUserTeams: function listUserTeams ({ id, organizationId, page = 1, limit }, cb) {
+      Joi.validate({ id, organizationId, page, limit }, validationRules.listUserTeams, function (err) {
+        if (err) return cb(Boom.badRequest(err))
+
+        const pageLimit = limit || _.get(config, 'authorization.defaultPageSize')
+        const offset = (page - 1) * pageLimit
+
+        const job = {
+          id,
+          organizationId,
+          offset,
+          limit: pageLimit,
+          user: {},
+          client: db
+        }
+
+        readUserTeams(job, (err) => {
+          if (err) return cb(err)
+          const pageSize = pageLimit || job.totalTeamsCount
```

Comment:
Why the or? Page size is either what the user wants or the default value. You don't want to return the total . Unless in last page if there are for example 10 teams and the page size is 25 you put Math.min(10,25).

Reply:
hold on with review, I'll review more carefully the code and push againa. Copy pasted from teams. Clearly this code is not ok, and also teams code is not ok.